app/services/idv/steps/document_capture_step.rb

```diff
@@ -43,11 +43,13 @@ def native_camera_ab_testing_variables
 
       def acuant_sdk_upgrade_a_b_testing_variables
         bucket = AbTests::ACUANT_SDK.bucket(flow_session[:document_capture_session_uuid])
-        acuant_version = (bucket == :use_newer_sdk) ? '11.7.1' : '11.7.0'
+        testing_enabled = IdentityConfig.store.idv_acuant_sdk_upgrade_a_b_testing_enabled
+        use_newer_sdk = (bucket == :use_newer_sdk)
```

Comment:
If testing is disabled, this will be `false` despite the fact we're using the newer of the two versions. Is that expected?

Reply:
What if instead of 'newer', we set default and alternate versions, and change the A/B test bucket to `use_alt_sdk`? Then it would be more explicit what the current default is meant to be, and we could test against an older version if we later discover what we suspect is a regression.
```
idv_acuant_sdk_version_default: <default_version>
idv_acuant_sdk_version_alternate: <version_to_test>
```

Reply:
Yeah I agree with @solipet -- "use newer" makes the logic more confusing. Having the two versions specified in the config also gives us fewer string replacements etc when changing Acuant versions for real. I'll whip something up